quickwit/quickwit-config/src/node_config/mod.rs

```diff
@@ -41,6 +42,14 @@ use crate::{ConfigFormat, MetastoreConfigs};
 
 pub const DEFAULT_QW_CONFIG_PATH: &str = "config/quickwit.yaml";
 
+#[derive(Clone, Debug, Eq, PartialEq, Serialize, Deserialize, Default)]
+#[serde(deny_unknown_fields)]
+pub struct RestConfig {
+    pub cors_allow_origins: Vec<String>,
+    #[serde(with = "http_serde::header_map")]
+    pub additional_headers: HeaderMap,
```

Comment:
GitHub code search fight votes for `extra`:

```suggestion
    pub extra_headers: HeaderMap,
```

Reply:
`extra`  +1